### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by kremalicious**
We all own the code by now 😀

We require approval by code owners on each PR so with this change everybody in this list is requested for review automatically on each PR ready for review. 

And with that, we can also think about bumping the requirement from 1 code owner to 2 code owner approvals for each PR as more eyes on each change are always preferable

### CODE REVIEW

**Review by mihaisc — APPROVED** [review]

**Comment by jamiehewitt15** [comment]
> And with that, we can also think about bumping the requirement from 1 code owner to 2 code owner approvals for each PR as more eyes on each change are always preferable

Requiring 2 code owner approvals seems like a good idea to me. It will ensure that we all get involved in the review process.

**Review by jamiehewitt15 — APPROVED** [review]

**Review by bogdanfazakas — APPROVED** [review]
The 2 code owner approvals sounds good since there are going to be 7 people notified 😃

**Pull request merged**
